libc/test/src/unistd/pipe2_test.cpp

```diff
@@ -0,0 +1,63 @@
+//===-- Unittests for pipe2
+//-------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/errno/libc_errno.h"
+#include "src/unistd/close.h"
+#include "src/unistd/pipe2.h"
+#include "src/unistd/read.h"
+#include "src/unistd/write.h"
+#include "test/UnitTest/ErrnoSetterMatcher.h"
+#include "test/UnitTest/Test.h"
+
+#include <fcntl.h>
```

Comment:
Please include include/llvm-libc-macros/linux/fcntl-macros.h instead.

Reply:
I need <fcntl.h> for [this](https://github.com/llvm/llvm-project/pull/85514/commits/863c2ab8fbc31c643fea421b150be4e04ada8a2b#diff-2c1e9264e6983d7340dc69520c94a83413e9b61ece4b1662ba5edfc46a66e230R43) subtest in Pipe2CreationTest.